src/main/java/org/jabref/gui/entryeditor/CitationRelationsTab.java

```diff
@@ -0,0 +1,628 @@
+    private CheckListView<CitationRelationItem> citedByListView;
+    private Button refreshCitingButton;
+    private Button refreshCitedByButton;
+    private Button importCitingButton;
+    private Button importCitedByButton;
+    private Button abortCitingButton;
+    private Button abortCitedButton;
+    private ProgressIndicator citingProgress;
+    private ProgressIndicator citedByProgress;
+    private final BibDatabaseContext databaseContext;
+    private final UndoManager undoManager;
+    private final StateManager stateManager;
+    private final FileUpdateMonitor fileUpdateMonitor;
+    private final PreferencesService preferencesService;
+    private final LibraryTab libraryTab;
+
+    //Tasks
+    private static BackgroundTask<List<BibEntry>> citingTask;
+    private static BackgroundTask<List<BibEntry>> citedByTask;
+
+    public static class CitationRelationItem {
+        private final BibEntry bibEntry;
+        private final boolean local;
+
+        public CitationRelationItem(BibEntry bibEntry, boolean local) {
+            this.bibEntry = bibEntry;
+            this.local = local;
+        }
+
+        public BibEntry getBibEntry() {
+            return bibEntry;
+        }
+
+        public boolean isLocal() {
+            return local;
+        }
+    }
+
+
+    public CitationRelationsTab(EntryEditorPreferences preferences, DialogService dialogService, BibDatabaseContext databaseContext, UndoManager undoManager, StateManager stateManager, FileUpdateMonitor fileUpdateMonitor, PreferencesService preferencesService, LibraryTab lTab) {
+        this.preferences = preferences;
+        this.dialogService = dialogService;
+        this.databaseContext = databaseContext;
+        this.undoManager = undoManager;
+        this.stateManager = stateManager;
+        this.fileUpdateMonitor = fileUpdateMonitor;
+        this.preferencesService = preferencesService;
+        this.libraryTab = lTab;
+        setText(Localization.lang("Citation relations"));
+        setTooltip(new Tooltip(Localization.lang("Show articles related by citation")));
+    }
+
+    /**
+     * The Pane that is shown when the functionality is not activated
+     *
+     * @param entry entry that is the context
+     * @return StackPane that is the activation screen
+     */
+    private StackPane getActivationPane(BibEntry entry) {
+        StackPane activation = new StackPane();
+        activation.setId("citation-relation-tab");
+        VBox alignment = new VBox();
+        alignment.setId("activation-alignment");
+        alignment.setFillWidth(true);
+        alignment.setAlignment(Pos.BASELINE_CENTER);
+        Label infoLabel = new Label(Localization.lang("The search is currently deactivated"));
+        Button activate = new Button(Localization.lang("Activate"));
+        activate.setOnAction(
+                event -> {
+                    JabRefPreferences prefs = JabRefPreferences.getInstance();
```

Comment:
please use the `PreferenceService`

Reply:
We implemented the activation according to the RelatedArticlesTab. We still use JabRefPreferences, because we couldn't figure out how to change the setting with the PreferenceService.

Reply:
Unfortunately the Related Articles tab did it wrong. We follow dependency infection principle, e.g. all dependencies are passed to the constructor. 
This allows us to mock them in tests etc 
And especially for the preferences. 
For the moment I would leave it as is, as we are currently working on restructuring the preferences/preferences service

Reply:
All Right so we will leave it as it is for the moment